GENETIS_doc/.gitignore

```
@@ -1,4 +1,5 @@
 __pycache__/
 .venv/
 *DS_Store
-docs/.vscode/
+*.vscode/
+*_build/
```

The `build` dir is still included, probably because of this:
```suggestion
*build/
```